doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -79,31 +79,34 @@ nRF9160: Asset Tracker v2
 
 * Added:
 
-   * Support for :ref:`Bosch Software Environmental Cluster (BSEC) library <bosch_software_environmental_cluster_library>` driver .
-   * Support for indoor air quality (IAQ) readings retrieved from the BME680 sensor on Thingy:91. For more information, see the :ref:`asset_tracker_v2_sensor_module`.
+   * Support for :ref:`Bosch Software Environmental Cluster (BSEC) library <bosch_software_environmental_cluster_library>` driver.
+   * Support for Indoor Air Quality (IAQ) readings retrieved from the BME680 sensor on Thingy:91. For more information, see the :ref:`asset_tracker_v2_sensor_module`.
    * Support for QEMU x86 emulation.
 
 * Updated:
 
   * For nRF Cloud builds, the configuration section in the shadow is now initialized during the cloud connection process.
   * Allow the :ref:`lib_nrf_cloud` library to handle modem FOTA updates if :kconfig:option:`CONFIG_NRF_CLOUD_FOTA` is enabled.
 
-nRF9160: Serial LTE Modem
+nRF9160: Serial LTE modem
 -------------------------
 
+* Added:
+
+  * #XDFUGET and #XDFURUN AT commands to support the cloud-to-nRF52 DFU service.
+
 * Updated:
 
   * Enhanced the #XHTTPCREQ AT command for better HTTP upload and download support.
   * Enhanced the #XSLEEP AT command to support data indication when idle.
   * Enhanced the MQTT client to support the reception of large PUBLISH payloads.
-  * Added the #XDFUGET and #XDFURUN AT commands to support the cloud-to-nRF52 DFU service.
 
 nRF Desktop
 -----------
 
 * Added:
 
-  * Documentation for selective HID report subscription in :ref:`nrf_desktop_usb_state` using :kconfig:option:`CONFIG_DESKTOP_USB_SELECTIVE_REPORT_SUBSCRIPTION` option.
+  * Documentation for selective HID report subscription in :ref:`nrf_desktop_usb_state` using :ref:`CONFIG_DESKTOP_USB_SELECTIVE_REPORT_SUBSCRIPTION <config_desktop_app_options>` option.
```

Comment:
I don't think we should have one-line entries to a section broken down like this. This is just adding unnecessary empty lines to the doc.

Reply:
Discussed with @greg-fer and  @FrancescoSer. Changed all such instances to one-line entries.